openfecwebapp/views.py

```diff
@@ -0,0 +1,70 @@
+from openfecwebapp.api_caller import load_totals
+from flask import render_template
+from openfecwebapp.data_mappings import (type_map, 
+map_candidate_table_values, map_committee_table_values, 
+map_candidate_page_values, map_totals, committee_type_map,
+generate_pagination_values)
+from werkzeug.exceptions import abort
+
+def render_search_results(results, query):
+    candidates = []
+    committees = []
+
+    if results.get('candidates') and results['candidates'].get(
+        'results'):
+        for c in results['candidates']['results']:
```

Comment:
Style question: which do you like more, the explicit guard or the implicit iteration?

```python
if results.get('candidates'):
    for c in results['candidates'].get('results', []):
```

You could take it a step further, but this seems ugly in Python
```python
for c in results.get('candidates', {}).get('results', []):
```